CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Unreleased
 ### Changed
 * Update the Firefox test browser to use the `selenium/node-firefox:3.141.59-titanium` docker image
 * Consume new docker images.
+* Bump axe-core to 3.4.0 and disable new rules introduced on axe-core v3.3 and v3.4 to remain passive. 
```

Comment:
can we list the rules we disabled for quick reference to consumers and for ourselves in the future.

Reply:
Good call. Updated. https://github.com/cerner/terra-toolkit/pull/369/commits/cd0e62d0227ab91e65ff1efbac1827f4b92c0341